input/intro-notes/StructureDefinition-au-core-medicationrequest-intro.md

```diff
@@ -24,5 +24,3 @@ Conformance in reverse is not guaranteed, i.e. a resource conforming to [Interna
   - Although both are marked as must support, servers are not required to support both a code and a reference, but they **SHALL** support *at least one* of these elements
   - A client application **SHALL** support both elements
   - When referencing a Medication resource, it is preferred the resource is [contained](http://hl7.org/fhir/R4/references.html#contained) but it may be an external resource
```

Comment:
Missing the profile-specific implementation guidance to treat reasonCode & reasonReference as a choice. See AU Core Procedure for an example.

Reply:
Done